### ISSUE TITLE
Invalid syntax error at filter flags expression

### ISSUE THREAD

**Issue by piorek94** [phaag/nfdump#255]
### Problem:

Issue is related to `nfdump` filter syntax. 

### Description:

Problem was detected for filter expression "flags AS" (probably conflicting with "as" expression for autonomous system numbers)

Valid expression "flags SA":
```
$ nfdump -r nfcapd.202010081633_anon -q -c 1 -o line "flags SA"
2020-10-08 16:32:48.640     0.000 TCP        28.40.65.88:443   ->   163.19.120.224:5609       500    26000     1
```
Equivalent expression (with reverse order of flags) "flags AS" throws an error:
```
$ nfdump -r nfcapd.202010081633_anon -q -c 1 -o line "flags AS"
Line 1: syntax error at 'AS'
```

### Expected behavior

Valid filter expressions should not throws error - should return proper data.

**Comment by phaag** [comment]
This is due to the fact, that 'AS' is recognised as a reserved word for the filter syntax `AS <number>`.
It needs an extra treatment.

**Comment by phaag** [comment]
Fixed in commit `b05764e`
